content/en/docs/toolbox/metering-and-billing/enable-billing.md

```diff
@@ -5,3 +5,59 @@ description: "Enable the billing function in KubeSphere to view the billing data
 linkTitle: "Enable Billing"
 weight: 15420
 ---
+
+This tutorial demonstrates how to enable KubeSphere Billing to view the cost of different resources in your cluster. By default, the Billing function is disabled so you need to manually add the price information in a ConfigMap.
+
+## Enable Billing
+
+1. Edit the ConfigMap `kubesphere-config` by running the following command:
+
+   ```bash
+   kubectl edit cm kubesphere-config -n kubesphere-system
+   ```
+
+2. Add the retention day and price information under `metering` in the ConfigMap.
+
+   | Parameter                                  | Description                                                  |
+   | ------------------------------------------ | ------------------------------------------------------------ |
+   | `retentionDay`                             | `retentionDay` determines the date range displayed on the **Metering and Billing** page for users. The value of this parameter must be the same as the value of `retention` in [Prometheus](../../../faq/observability/monitoring/#how-to-change-the-monitoring-data-retention-period). |
+   | `currencyUnit`                             | The currency that displays on the **Metering and Billing** page. Currently allowed values are `CNY` (Renminbi) and `USD` (US dollars). If you specify other currencies, the console will display cost in CNY by default. |
```

Comment:
There is a type error in ZH doc. We should change "If you specify other currencies, the console will display cost in CNY by default." to "If you specify other currencies, the console will display cost in USD by default."

Reply:
This is related with https://github.com/kubesphere/website/pull/1407

Reply:
Fixed.